src/amo/reducers/users.js

```diff
@@ -423,17 +423,50 @@ export const unsubscribeNotification = ({
   };
 };
 
-export const getUserById = (users: UsersState, userId: UserId) => {
+export const getUserById = (users: UsersState, userId: UserId): UserType => {
   invariant(typeof userId === 'number', 'userId is required');
   return users.byID[userId];
 };
 
-export const getUserByUsername = (users: UsersState, username: string) => {
+export const getUserByUsername = (
+  users: UsersState,
+  username: string,
+): UserType => {
   invariant(username, 'username is required');
   return users.byID[users.byUsername[username.toLowerCase()]];
 };
 
-export const getCurrentUser = (users: UsersState) => {
+export const getCurrentUser = (
+  users: UsersState,
+): null | {|
```

Comment:
Don't e have a `UserType` or `InternalUserType`?

Reply:
Yes, and the codemod used it in most places, but for some reason not here. I've fixed it.